#! /bin/sh /usr/share/dpatch/dpatch-run
## cupsfilter-path-typo.dpatch by Johan Kiviniemi <debian@johan.kiviniemi.name>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix a typo in cupsfilter, causing filters not to have /bin in their
## DP: PATH.

@DPATCH@
diff -urNad cups-1.3.8~/scheduler/cupsfilter.c cups-1.3.8/scheduler/cupsfilter.c
--- cups-1.3.8~/scheduler/cupsfilter.c	2008-08-13 12:14:17.803302953 +0300
+++ cups-1.3.8/scheduler/cupsfilter.c	2008-08-13 12:14:36.815297551 +0300
@@ -928,7 +928,7 @@
   }
 
   snprintf(line, sizeof(line),
-           "%s/filter:" CUPS_BINDIR ":" CUPS_SBINDIR ":/bin/usr/bin",
+           "%s/filter:" CUPS_BINDIR ":" CUPS_SBINDIR ":/bin:/usr/bin",
 	   ServerBin);
   set_string(&Path, line);
 
